redesign/team.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="description" content="Find out more about our core product development team at dwyl, based in East London, making your dreams come true">
```

Comment:
Find out more about our core product development team at dwyl, based in East London, using technology to solve real problems